### ISSUE TITLE
Recording problem in eu-south-1 and af-south-1

### ISSUE THREAD

**Issue by vknaisl** [aws-samples/amazon-chime-sdk#187]
**Describe the bug**
Even though we turn on the region and wait for a couple of days, we still can't use media pipelines to record chime meetings in these two regions (`eu-south-1` and `af-south-1`). 

**To Reproduce**
Even though we successfully turn on the recording, we don't receive any content in the S3 bucket in the same way as we got the content in the different regions. However, the recording pretends to be successful. We also receive events through the event bridge about the recording. So everything seems to be successful. But there is no content in the S3 bucket.

**Comment by simmkyu** [comment]
In the [Chime SDK Available regions guide](https://docs.aws.amazon.com/chime-sdk/latest/dg/sdk-available-regions.html), `eu-south-1` and `af-south-1` are marked with `*`.
> Regions marked with an asterisk (*) must be enabled in your AWS account. For more information, refer to [Enabling a Region](https://docs.aws.amazon.com/general/latest/gr/rande-manage.html) in the AWS General Reference.

Have you enabled these two regions in the AWS console? If not, you can follow the steps:
https://docs.aws.amazon.com/general/latest/gr/rande-manage.html#rande-manage-enable

**Comment by vknaisl** [comment]
Yes, we enabled it and even waited for a few days. And still, it doesn't work

**Comment by ajselvar-work** [comment]
Hello @vknaisl, Can you provide a sample requestId or invoke the API and provide the request body for us to further debug?

**Comment by sinanliuAmazon** [comment]
Hi @vknaisl, our log shows `IllegalLocationConstraintException`, could you please double check if the S3 bucket provided in CreateMediaCapturePipeline request belongs to the same AWS Region as the Amazon Chime SDK meeting? https://docs.aws.amazon.com/chime-sdk/latest/dg/create-s3-bucket.html

**Comment by vknaisl** [comment]
Hi,

No, the meeting is in the region `eu-south-1`. However, the S3 bucket is in `us-east-1` region. We didn't know that. Nevertheless, for the rest of the regions except `eu-south-1` and `af-south-1`, it works. Can you please explain me why it works?

Thanks!

Vojtech

**Comment by sinanliuAmazon** [comment]
Hi @vknaisl , Customer will see failures when 
- meeting is in manual-enabled region while S3 bucket is in a different region
- S3 bucket is in manual-enabled region while meeting is in a different region

Because S3 has limitation on the manual-enabled region, either of above scenario will cause a failed S3 operation failure, so in the developer doc, we ask customer to make the bucket and meeting in the same region to avoid failure.

**Comment by vknaisl** [comment]
Hi,

Thanks for the explanation. We created a bucket for each region and it helped :)

Vojtech
